editor/common/src/EditorProps.ts

```diff
@@ -0,0 +1,45 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Editing
+ */
+
+/** @internal */
+export const editCommandApi = {

```

Comment:
No docs for any of these types?

Reply:
I'm not going to bother to document any of this until Brien and I spend more time fleshing it out. The only reason it's in this PR is to get the package.json files included so I don't have to edit them every time I merge. I started working on this and realized that I couldn't make it work until I straightened out BriefcaseDb.

Do you think it'd be clearer that none of this is stable if I marked it @alpha rather than @internal?

Reply:
Fair enough. Yes, `internal` for APIs we don't intend others to use *ever*; `alpha` for APIs we don't intend others to use *yet*.